src/request/remove-key/index.html

```diff
@@ -5,14 +5,14 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <meta http-equiv="Content-Security-Policy"
-          content="default-src 'self' 'unsafe-eval' https://cdn.nimiq-testnet.com;
+          content="default-src 'self' 'unsafe-eval' https://cdn.nimiq-network.com;
```

Comment:
Why those changes?

Reply:
Because this branch has to use the master branch if the core repo, which is not yet released on the testnet CDN. Only on the CI-enabled nimiq-network CDN.

Reply:
Ok. Then you should change the string to be replaced in `/tools/build.sh` as well, I guess.

Reply:
...or do you want to change it back after release of core?

Reply:
Yes, I'm changing it back to nimiq-testnet I think after core release.